tests/test_unit.py

```diff
@@ -54,3 +55,64 @@ def test_repeating_parameters(add_duplicate):
         ]
     else:
         assert instrument.get_par_by_name("duplicate-name") == p1
+
+
+@pytest.mark.parametrize("value",  [25, 25., 25.64547871216879451687311211245117852145229614585985498212321, "aaaa"])
+def test_float_defaults(value):
+    if type(value) == str:
+        with pytest.raises(RuntimeError):
+            Float(
+                value=value,
+                name="p_float"
+            )
+    else:
+        p_float = Float(
+            value=value,
+            name="p_float"
+        )
+
+        assert p_float.get_value_in_default_format(p_float.value) == p_float.value
```

Comment:
`value` instead of `p_float.value`? The latter possibly already transformed.

Reply:
I dont understand, what would you change?

Reply:
like you did in another case:

```suggestion
        assert p_float.get_value_in_default_format(p_float.value) == value
```